### DISCUSSION TITLE
Duda sobre front

### DISCUSSION THREAD

**Discussion by gersondlarosa** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#449] [Q&A] [answered]
Buen día compañeros alguien me podría indica como hacer para que se ponga a un ladito mi texto y no por debajo al hacer la acción de busca

![poke dex](https://user-images.githubusercontent.com/68626555/159053996-6a8f32e1-2750-4508-9194-4ff50091a469.png)
r

**Comment by manusanabria** [comment]
Revisa el tema de flexbox para que puedas cambiar la posición https://developer.mozilla.org/es/docs/Web/CSS/CSS_Flexible_Box_Layout/Basic_Concepts_of_Flexbox

**Reply by gersondlarosa** [reply]
En si según yo es más un tema de js por que el html lo pongo a si directo del id lo pone abajito



y el JS lo mando a llamar con este 

const orderPokemon = (url) => {
   const order = document.getElementById("order")
         order.innerHTML = url;
}

y si yo pongo el div a si me quita lo que esta adentro
![div](https://user-images.githubusercontent.com/68626555/159056440-051e88e2-12fd-4f31-bab9-e47f3a56ddab.png)

**Comment by gersondlarosa** [comment]
Cuando lo mando a si solo me muestra los datos y me pita el Hp: o lo que esta antes
![borrado](https://user-images.githubusercontent.com/68626555/159056868-2128a9ce-d2ba-491f-8720-02ab5da0f62e.png)

**Comment by PlayerOneV** [comment, marked as answer]
Puedes intentar recuperando el nombre del stat y el valor (name, base_stat) en js (cada uno en su respectiva variable) y los puedes concatenar para que estén juntos y con eso solamente los pondrías en el div donde los vayas a mostrar.
![image](https://user-images.githubusercontent.com/75816087/159062708-5c194023-bf8b-47fb-93de-2c2a130f164e.png)
